src/pudl/package_data/eia861/column_maps/short_form_eia861.csv

```diff
@@ -4,13 +4,13 @@ utility_id_eia,1,1,1,1,1,1,1,-1,1,1,1
 utility_name_eia,2,2,2,2,2,2,2,-1,2,2,2
 entity_type,-1,-1,-1,3,3,3,3,-1,3,3,3
 state,3,3,3,4,4,4,4,-1,4,4,4
-ba_code,-1,4,4,5,5,5,5,-1,5,5,5
-total_revenue,4,5,5,6,6,6,6,-1,6,6,6
-total_sales,5,6,6,7,7,7,7,-1,7,7,7
-total_customers,6,7,7,8,8,8,8,-1,8,8,8
-water_heater,-1,8,8,9,9,9,9,-1,9,9,9
-net_metering,8,9,9,10,10,10,10,-1,10,10,10
-demand_side_management,9,10,10,11,11,11,11,-1,11,11,11
-time_based_programs,10,11,11,12,12,12,12,-1,12,12,12
-green_pricing,7,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
+balancing_authority_code_eia,-1,4,4,5,5,5,5,-1,5,5,5
+sales_revenue,4,5,5,6,6,6,6,-1,6,6,6
+sales_mwh,5,6,6,7,7,7,7,-1,7,7,7
+customers,6,7,7,8,8,8,8,-1,8,8,8
```

Comment:
Maybe this would have consequences for other tables and so we don't want to deal with it right now, but I think this should probably be `num_customers`

Reply:
In the src.pudl.metadata.fields, there's the customers key in the JSON fields. There's no num_customers or total_customers. Please check to confirm that this is correct so I'll know if I still need to change this.

Reply:
What do you mean by "customers key"?

Reply:
@Nancy9ice what I meant here is that I think **we** originally did a poor job of naming this column, or that it was named before we had thought about [our naming conventions](https://catalystcoop-pudl.readthedocs.io/en/nightly/dev/naming_conventions.html#general-abbreviations), so I was suggesting that we take this opportunity to rename `customers` which is pretty vague, to something that provides a bit more context about the column, like `customers_num` or `num_customers` though searching through `fields.py` for other instances of `_num` and `num_` it doesn't seem as if we have established a strong convention right now, so maybe we should just ignore this for now and deal with it in a future round of renaming / deeper EIA-861 integration.